things/postgres/things.go

```diff
@@ -89,10 +89,7 @@ func (tr thingRepository) RetrieveByID(owner, id string) (things.Thing, error) {
 
 	if err != nil {
 		empty := things.Thing{}
-		if err == sql.ErrNoRows {
-			return empty, things.ErrNotFound
-		}
-		return empty, err
+		return empty, things.ErrNotFound
```

Comment:
If we do this, we will ignore every other error (like connection errors). I think that this removed `if` should stay. Maybe problem isn't in this part of code.